### PR TITLE
populate viewer data in konnekt search

### PR DESCRIPTION
**Pull request by killbotXD**
Screenshot from 2019-09-15 18-20-44

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "core-js": "^2.6.5",
     "register-service-worker": "^1.6.2",
     "vue": "^2.6.10",
+    "vue-apollo": "^3.0.0-beta.11",
```
Comment (ajatprabha) [inline package.json]: Can we use the latest stable version instead of beta?

File: src/components/Toolbar.vue [hunk 2]
```diff
@@ -18,7 +18,7 @@
             span(v-if="$vuetify.breakpoint.mdAndUp") Account
             v-icon(v-if="$vuetify.breakpoint.mdAndUp") mdi-menu-down
         v-list
-          v-list-tile(v-for="(item,i) in accountItems" :key="i")
+          v-list-tile(v-for="(item,i) in AccountItems" :key="i")
```
Comment (ajatprabha) [inline src/components/Toolbar.vue]: Should start with lower case.

File: src/components/KonnektSearch.vue [hunk 3]
```diff
@@ -57,12 +58,42 @@ class Person {
...
+    `
+  },
   name: "KonnektSearch",
   components: { Toolbar },
   data: function() {
     return {
-      search: "",
+      viewer: {},
+      search: {},
+      search_1: "",
```
Comment (ajatprabha) [inline src/components/KonnektSearch.vue]: Any other meaningful name?

File: src/components/KonnektSearch.vue [hunk 4]
```diff
@@ -5,20 +5,20 @@
...
               prepend-inner-icon="mdi-magnify"
               label="search any skill or name"
               clearable)
           v-flex
-            v-btn(large color="primary") GO
-    v-container
-      div(v-for="person in filteredList" v-bind:key="person.name")
+            v-btn(large color="primary" to="") GO
+    v-container(v-if="!$apollo.loading")
+      div(v-for="(node,i) in search.edges" :key="i")
```
Comment (ajatprabha) [inline src/components/KonnektSearch.vue]: Check if `({ node }, i) in search.edges` work, you'll not have to do `node.node` in many places then.

File: package.json (hunk 1 above)
Comment (killbotXD) [reply package.json]: Yes we can.should I change the `package.json`?

File: src/components/Toolbar.vue (hunk 2 above)
Comment (killbotXD) [reply src/components/Toolbar.vue]: Apologies I don't know why I changed it to uppercase.

File: src/components/KonnektSearch.vue (hunk 3 above)
Comment (killbotXD) [reply src/components/KonnektSearch.vue]: how about `search_query`?

File: src/components/KonnektSearch.vue (hunk 4 above)
Comment (killbotXD) [reply src/components/KonnektSearch.vue]: works nicely. changing to {node} instead of node.

Comment (killbotXD) [reply src/components/KonnektSearch.vue]: done :)

File: src/components/KonnektSearch.vue (hunk 3 above)
Comment (killbotXD) [reply src/components/KonnektSearch.vue]: changed name to search query

**Pull request merged**